benchmarks/src/main/scala/akka.kafka.benchmarks/ReactiveKafkaConsumerBenchmarks.scala

```diff
@@ -69,7 +69,7 @@ object ReactiveKafkaConsumerBenchmarks extends LazyLogging {
         batch.updated(elem)
 
       }
-      .mapAsync(1) { m =>
+      .mapAsync(3) { m =>
```

Comment:
The Commit message to the actor will arrive in order, which means that the `consumer.commitAsync` calls are done in order so I don't see where they can be unordered. We even call `poll` for each received `Commit`.